src/splitInteger.js

```diff
@@ -1,9 +1,12 @@
+'use strict';
+
 /**
  * @param {number} value
  * @param {number} numberOfParts
  *
  * @returns {number[]}
  */
+
 function splitInteger(value, numberOfParts) {
```

Comment:
The function `splitInteger` is missing the implementation to split the integer and return the required array. You can start by dividing the value by the numberOfParts and then distribute the remaining value among the elements to maintain the difference between max and min values. 🧐